src/FormHandler/HandlerConfigInterface.php

```diff
@@ -52,6 +52,18 @@ public function onSuccess(callable $callback);
     public function onFailure(callable $callback);
 
     /**
+     * Register the FormSubmitProcessor
+     *
+     * This will overwrite the default FormSubmitProcessor.
+     *
+     * signature:
+     * function (FormInterface $form, Request $request): void;
+     *
+     * @param callable $callback
+     */
+    public function setFormSubmitProcessor(callable $callback);
```

Comment:
Can you document this as BC break and mark the interface and class as `@internal`? This class is not meant to be implemented by user-land and only serves as possibility to mock/prophesize.

While officially we follow semver and changing a non-internal interface is a BC break, I anticipate that this interface is not widely implemented as it would serve no purpose.